test.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_s3_bucket" "acme-s3-access-logging" {
+  bucket = "test"
+}
+
+resource "aws_s3_bucket" "not-another-public-one-please" {
+  bucket = "my-public-bucket"
+  acl    = "public-read"
+
+  logging {
+    target_bucket = data.aws_s3_bucket.acme-s3-access-logging.id
+    target_prefix = "my-passing-bucket/logs/"
+  }
+
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm = "AES256"
+      }
+    }
+  }
+}
```

Comment:
## result
tfsec check aws-s3-specify-public-access-block failed.
## severity
⚠️MEDIUM
## reason
Resource aws_s3_bucket.not-another-public-one-please has no associated aws_s3_bucket_public_access_block.
## how to ignore
`#tfsec:ignore:AWS098`([refs](https://github.com/aquasecurity/tfsec#ignoring-warnings))

For more information, [see](https://tfsec.dev/docs/aws/s3/specify-public-access-block#aws/s3)